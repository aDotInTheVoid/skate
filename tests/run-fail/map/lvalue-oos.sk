fn main() {
    x.y = z;
}