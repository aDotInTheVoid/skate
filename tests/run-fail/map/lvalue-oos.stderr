error: Couldn't find variable `x` in scope
  ┌─ $DD/tests/run-fail/map/lvalue-oos.sk:2:5
  │
2 │     x.y = z;
  │     ^
  │
  = Variables in scope: []

